webapp/src/pages/ComponentsBook/ComponentsBook.tsx

```diff
@@ -0,0 +1,34 @@
+import { Text, TextColor, TextVariant, TextWeight } from '../../shared/components';
+import './ComponentsBook.css';
+import {
+  NavigationExample,
+  TextExample,
+  ButtonsExample,
+  StatusExample,
+  IconExample,
+  AvatarExample,
+  RowExample,
+  InputExample,
+} from './ComponentsExamples';
+
+export default function ComponentsBook() {
```

Comment:
When you want to add a new example, just create a new `NameOfYourComponentExample` with the showcase of it and add it here